strawberry/chalice/views.py

```diff
@@ -76,7 +78,7 @@ def render_graphiql(self, request: Request) -> Response:
         Returns:
             The GraphiQL html page as a string
         """
-        return get_graphiql_html(subscription_enabled=False)  # type: ignore
+        return get_graphiql_html(subscription_enabled=False, example_query=self.graphiql_example_query)  # type: ignore
```

Comment:
Same as above:
Should this param be only named `example_query`?

Reply:
Yes, I will update it